Sources/SQLiteKit/SQLiteDataEncoder.swift

```diff
@@ -33,107 +30,54 @@ public struct SQLiteDataEncoder {
     }
 
     private final class _Encoder: Encoder {
-        var codingPath: [CodingKey] {
-            return []
-        }
-
-        var userInfo: [CodingUserInfoKey : Any] {
-            return [:]
-        }
-
+        var codingPath: [any CodingKey] = []
```

Comment:
why the change from computed to stored? 🤔

Reply:
1) Because these properties shouldn't really be hardcoded this way and this is preparatory to a more substantial improvement to how this crud works.
2) As stored properties the fact that they're never used (even by Codable itself) doesn't count against the test coverage percentage 😂